app/javascript/types/responses/ItemCreateResponse.ts

```diff
@@ -0,0 +1,32 @@
+// This is a generated file. Do not edit this file directly.
```

Comment:
The JSON schemas are pretty verbose, as it is, so it's an extra bummer that each one gets committed to the repo twice (once as a TypeScript file, as seen here, and once as a JSON file, which is copied into the corresponding TypeScript file). As mentioned [here](https://github.com/ThomasAribart/json-schema-to-ts/blob/e9ac5ee/documentation/FAQs/does-json-schema-to-ts-work-on-json-file-schemas.md), if this issue is ever addressed, then I guess we'll be able to remove this duplication: https://github.com/microsoft/TypeScript/issues/ 32063 .